bootstrap/cmd/bootstrap/app/ksServer.go

```diff
@@ -45,15 +48,18 @@ const CACHED_REGISTRIES = "/opt/versioned_registries"
 // key used for storing start time of a request to deploy in the request contexts
 const START_TIME = "startTime"
 
+const KUBEFLOW_REG_NAME = "kubeflow"
+const DM_FOLDER  = "deploymentmanager"
```

Comment:
Can we use the same folder layout that kfctl produces?

So we should have the directories:
gcp_config
k8s_specs
ks_app/